libs/dataset-explorer/src/lib/TableView/TableView.tsx

```diff
@@ -0,0 +1,294 @@
+import React from "react";
+
+import { tableViewStyles } from "./TableView.styles";
+import { ITableViewProps } from "./TableViewProps";
+import { IITableViewState, ITableViewTableState } from "./TableViewState";
+
+export class TableView extends React.Component<
+  ITableViewProps,
+  IITableViewState
+> {
+  public static contextType = ModelAssessmentContext;
+  public context: React.ContextType<typeof ModelAssessmentContext> =
+    defaultModelAssessmentContext;
+  private readonly maxSelectable = 5;
+
+  private selection: Selection = new Selection({
+    onSelectionChanged: (): void => {
+      const c = this.selection.getSelectedCount();
+      const indices = this.selection.getSelectedIndices();
+      if (c === this.maxSelectable) {
+        this.setState({ selectedIndices: indices });
+      }
+      if (c > this.maxSelectable) {
+        for (const index of indices) {
+          if (!this.state.selectedIndices.includes(index)) {
+            this.setState({ indexToUnselect: index });
+          }
+        }
+      }
+      this.props.onAllSelectedItemsChange(this.selection.getSelection());
+      this.props.telemetryHook?.({
+        level: TelemetryLevels.ButtonClick,
+        type: TelemetryEventName.IndividualFeatureImportanceSelectedDatapointsUpdated
+      });
+    }
+  });
+
+  public constructor(props: ITableViewProps) {
+    super(props);
+
+    const tableState = this.updateItems();
+
+    this.state = {
+      indexToUnselect: undefined,
+      selectedIndices: [],
+      ...tableState
+    };
+  }
+
+  public componentDidUpdate(prevProps: ITableViewProps): void {
+    if (this.props.selectedCohort !== prevProps.selectedCohort) {
+      this.setState(this.updateItems());
+    }
+    if (this.state.indexToUnselect) {
+      this.selection.toggleIndexSelected(this.state.indexToUnselect);
+      this.setState({ indexToUnselect: undefined });
+    }
+  }
+
+  public render(): React.ReactNode {
+    if (this.state.rows === undefined || this.state.columns === undefined) {
+      return React.Fragment;
+    }
+    const hasTextImportances =
+      !!this.context.modelExplanationData?.precomputedExplanations
+        ?.textFeatureImportance;
+    const classNames = tableViewStyles();
+    return (
+      <Stack tokens={{ padding: "l1" }}>
+        <Stack.Item className={classNames.infoWithText}>
+          <Text variant="medium">
+            {localization.ModelAssessment.FeatureImportances.IndividualFeature}
+          </Text>
+        </Stack.Item>
+        <Stack.Item className={classNames.selectionCounter}>
+          <LabelWithCallout
+            label={localization.formatString(
+              localization.ModelAssessment.FeatureImportances.SelectionCounter,
+              this.selection.count,
+              this.maxSelectable
+            )}
+            calloutTitle={undefined}
+            renderOnNewLayer
+            type="label"
+          >
+            <Text block>
```

Comment:
Can we not set text with block? with block,  when making screen too small , the overlfow part of the text will not be seen.

Reply:
Could you please try to change the screen size to 320px? If the text is still visible, we should be fine

Reply:
removed block, thanks!